examples/object-oriented-cpp/Makefile

```diff
@@ -0,0 +1,12 @@
+CXX = c++ -std=c++11
```

Comment:
Usually this is just the binary, flags you put in `CXXFLAGS`

Reply:
That depends. The flag effectively changes the compiler, so it should go there. See also the [autoconf macro](https://www.gnu.org/software/autoconf-archive/ax_cxx_compile_stdcxx_11.html#ax_cxx_compile_stdcxx_11).

Reply:
So should I revert?

Reply:
It probably doesn't matter at this point.

Reply:
Fine